### PR TITLE
return more info for query device

### PR DESCRIPTION
**Pull request by dougperez**
- bump version

### CODE REVIEW

File: lib/conduit/sprint/parsers/query_device_info.rb [hunk 1]
```diff
@@ -14,12 +14,68 @@ class QueryDeviceInfo::Parser < Parser::Base
...
+
+    attribute :device_type do
+      content_for '//deviceType'
+    end
+
+    attribute :device_type do
+      content_for '//deviceType'
+    end
+
+    attribute :device_type do
```
Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Why so many device_type attributes?

File: lib/conduit/sprint/parsers/query_device_info.rb [hunk 2]
```diff
@@ -14,12 +14,68 @@ class QueryDeviceInfo::Parser < Parser::Base
...
+
+    attribute :equipment_freq_type_code do
+      content_for '//equipmentFreqTypeCode'
+    end
+
+    attribute :uicc_not_available_reason_code do
+      content_for '//uiccNotAvailableReasonCode'
+    end
+
+    attribute :not_available_reason_code do
```
Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Do we translate these anywhere?

File: lib/conduit/sprint/parsers/query_device_info.rb (hunk 1 above)
Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Missed this, thanks.

File: lib/conduit/sprint/parsers/query_device_info.rb (hunk 2 above)
Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: We do not translate at the moment, but could see us translating here when we have the mapping.

Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Should be mapping in the documentation.  Do you want me to send it to you?

Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: @pedromartinez translation added.

**Pull request merged**